Office365-Admin/add-users/get-access-to-and-back-up-a-former-user-s-data.md

```diff
@@ -29,62 +29,59 @@ description: "Learn how to preserve an employee's files and emails when the pers
-    
-3. Go to **Active Users** and select the user.
-    
-4. Expand **OneDrive Settings** in the user details pane, and select **Access files**.
-    
-5. Copy the files to your own OneDrive for Business or a common location.
-    
-    There are a few ways to copy files in Office 365. See [Video: Set up document storage and sharing in Office 365](https://support.office.com/article/6a41fb61-9c11-4adf-9b98-f7910fe1b0d2.aspx). Or, to sync your files, and then upload those files to your OneDrive for Business or your team site, see [Sync files with the new OneDrive sync client in Windows](https://support.office.com/article/615391c4-2bd3-4aae-a42a-858262e42a49.aspx).
+1. In the admin center, go to the **User management > [Active users](https://go.microsoft.com/fwlink/p/?linkid=834822)** page.
     
+2. Select a user.
+
+3. In the right page, choose  **OneDrive**. Under **Get access to files**, choose **Create link to files**.
+
+4. Click the link to open the file location and download and copy the files to your own OneDrive for Business, or a common location. You can also share the link with another user to download the files.
+
     > [!NOTE]
     > You can move up to 500 MB of files and folders at a time.
     > When you use **Move to** with documents that have version history, only the latest version is moved. To move earlier versions, you need to restore and move each one. 
     
 ## Revoke admin access to a user’s OneDrive site
 
+As global admin you have access to the content in a user’s OneDrive site, but you may want to remove your access to a user’s documents. By default, the OneDrive Site Collection Administrator is the owner of the OneDrive account. The following steps describe how to remove a Site Collection Admin from a user’s OneDrive site.
 
-As global admin you have access to the content in a user’s OneDrive site, but you may want to remove your access to a user’s documents. By default, the OneDrive Site Collection Administrator is the owner of the OneDrive account. The following steps describe how to remove a Site Collection Admin on a user’s OneDrive site.
-
-1. Sign in to the [![Office 365 admin center.](../media/e00ba917-c3fb-4173-b344-43eb5c7eeb15.png)](https://portal.office.com/adminportal/home) as a global or SharePoint admin. 
+1. Sign in to the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=837890) as a global admin or SharePoint admin. 
 
- > [!NOTE]
- > If you see a message that you don't have permission to access the page, you don't have Office 365 administrator permissions in your organization.)
+    If you get a message that you don't have permission to access the admin center, then you don't have administrator permissions in your organization.
 
-2. In the left pane, under **Admin Centers**, select **SharePoint**.
+2. In the left pane, choose **Admin centers > SharePoint**.
```

Comment:
"Admin centers" and "SharePoint" should be individually bold, but not the > in between:
**Admin centers** > **SharePoint**